__test__/FilterSidebarCheckboxFacetGroup.test.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
```

Comment:
This file should be named after the component it’s testing: `CheckboxFacetGroup.test.js`.